packages/esm-patient-chart-app/src/visit/visits-widget/visit-context/visit-context-header.component.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button, Loading } from '@carbon/react';
+import { showModal, useVisit } from '@openmrs/esm-framework';
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './visit-context-header.scss';
+import VisitContextInfo from './visit-context-info.component';
+import { useVisitContextStore } from './visit-context';
+
+interface VisitContextHeaderProps {
+  patientUuid: string;
+}
+
+const VisitContextHeader: React.FC<VisitContextHeaderProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+  const { currentVisit, isLoading } = useVisit(patientUuid);
+  const { manuallySetVisitUuid, setVisitContext } = useVisitContextStore();
+  const isActiveVisit = !Boolean(currentVisit && currentVisit.stopDatetime);
```

Comment:
It occurs to me that there's likely to be confusion where people think current visit = active visit (see my confusion above) though a better name hasn't come to mind for me yet.